WISE-Community/WISE
# Clean up java

I removed a bunch of redundant comments and fixed some code formatting issues. No new functionality was introduced. However, since I had worked on this in January 2018, I had to resolve a bunch of conflicts when trying to merge into PortalRedesignAngular and could have introduced new bugs.

To test:
1. run "mvn test" and make sure that tests pass
2. Do a cursory test of major parts of WISE
